admin-dev/themes/new-theme/js/pages/order/create/shipping-renderer.js

```diff
@@ -84,6 +86,36 @@ export default class ShippingRenderer {
     });
   }
 
+  /**
+   * @param useRecycledPackaging
+   *
+   * @private
+   */
+  _renderRecycledPackagingSwitch(useRecycledPackaging) {
+    $(createOrderMap.recycledPackagingSwitch).each((key, input) => {
+      if (input.value === '1') {
+        input.checked = useRecycledPackaging;
+      } else {
+        input.checked = !useRecycledPackaging;
+      }
+    });
+  }
+
+  /**
+   * @param isAGift
+   *
+   * @private
+   */
+  _renderGiftSwitch(isAGift) {
```

Comment:
```suggestion
  renderGiftSwitch(isAGift) {
```

Reply:
On 177 and in this file, we use `_` everywhere (on develop it's fixed as eslint screams at us if we do that), can I keep it here to stay coherent with the rest of the file ?

Reply:
It will be easier for the merge into the develop branch. We already have files with mixed style in the 1.7.7.x.